cli/config_test.go

```diff
@@ -91,9 +91,10 @@ func TestIsSet(t *testing.T) {
 }
 
 func TestGet(t *testing.T) {
-	type args struct {
-		key string
-	}
+	// TODO: why not used?
```

Comment:
auto generated by vscode plugin, can be deleted

Reply:
removed